core/sql/sqlcomp/CmpSeabaseDDLcommon.cpp

```diff
@@ -10991,7 +10997,20 @@ CmpSeabaseDDL::setupHbaseOptions(ElemDDLHbaseOptions * hbaseOptionsClause,
           hbaseOptionsStr += optionStr;
         }
     }
+  if (!flushSize.isNull() && !memstoreFlushSizeOptionSpecified)
+    {
+      hbaseOption = new(STMTHEAP) HbaseCreateOption("MEMSTORE_FLUSH_SIZE", 
+                                                    flushSize.data());
+      hbaseCreateOptions.insert(hbaseOption);
 
+      if (ActiveSchemaDB()->getDefaults().userDefault
+          (HBASE_MEMSTORE_FLUSH_SIZE_OPTION) == TRUE)
+        {
+          numHbaseOptions += 1;
+          sprintf(optionStr, "MEMSTORE_FLUSH_SIZE='%s'|", flushSize.data());
```

Comment:
I would recommend using snprintf instead of sprintf, to avoid the possibility of a buffer overrun (probably not likely here, but just in case).

```
snprintf(optionStr, sizeof(optionStr), ...)
```

Reply:
I agree, I can change this into a more robust coding style, using snprintf.